script/verification/ProtocolVersionsBump.s.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {console2 as console} from "forge-std/console2.sol";
+import {Vm} from "forge-std/Vm.sol";
+import {LibString} from "solady/utils/LibString.sol";
+import {VerificationBase, SuperchainRegistry} from "script/verification/Verification.s.sol";
+import "@eth-optimism-bedrock/src/dispute/lib/Types.sol";
+import {FaultDisputeGame} from "@eth-optimism-bedrock/src/dispute/FaultDisputeGame.sol";
+import {PermissionedDisputeGame} from "@eth-optimism-bedrock/src/dispute/PermissionedDisputeGame.sol";
+import {DisputeGameFactory} from "@eth-optimism-bedrock/src/dispute/DisputeGameFactory.sol";
+import {MIPS} from "@eth-optimism-bedrock/src/cannon/MIPS.sol";
+import {ISemver} from "@eth-optimism-bedrock/src/universal/ISemver.sol";
+import {Simulation} from "@base-contracts/script/universal/Simulation.sol";
+import {NestedMultisigBuilder} from "@base-contracts/script/universal/NestedMultisigBuilder.sol";
+import {ProtocolVersions, ProtocolVersion} from "@eth-optimism-bedrock/src/L1/ProtocolVersions.sol";
+import {LibString} from "solady/utils/LibString.sol";
+
+abstract contract ProtocolVersionsBump is VerificationBase, SuperchainRegistry {
+    struct ProtoVer {
+        uint32 major;
+        uint32 minor;
+        uint32 patch;
+        uint32 preRelease;
+    }
+
+    uint256 immutable reccomended;
+    uint256 immutable required;
+
+    address owner;
+
+    constructor(address _owner, ProtoVer memory _reccomended, ProtoVer memory _required) {
+        owner = _owner;
+        reccomended = encodeProtocolVersion(_reccomended);
+        required = encodeProtocolVersion(_required);
+        console.log(owner);
```

Comment:
Seems to be a log for testing this should be removed or we should add the more information like `console.log("Current Owner Safe is:", owner);`

Reply:
Done